hollow/src/main/java/com/netflix/hollow/api/consumer/metrics/ConsumerRefreshMetrics.java

```diff
@@ -73,12 +74,15 @@ public UpdatePlanDetails getUpdatePlanDetails() {
     public long getConsecutiveFailures() {
         return consecutiveFailures;
     }
-    public OptionalLong getRefreshSuccessAgeMillisOptional() {
-        return refreshSuccessAgeMillisOptional;
+    public OptionalLong getRefreshSuccessAgeMillis() {
```

Comment:
This is also potentially a breaking change

Reply:
Yes, I can back this one out. The gain is minimal, probably not worth it. Thanks for catching it.